client/src/components/MapView.vue

```diff
@@ -93,10 +94,10 @@ export default defineComponent({
         this.popup.remove();
       }
 
-      if (newDataLayer != null && !this.isDataLayerVisible(newDataLayer.styleLayer.id)) {
+      if (newDataLayer != null) {
         this.setDataLayerVisibility(newDataLayer.styleLayer.id, true);
       }
-      if (oldDataLayer != null && this.isDataLayerVisible(oldDataLayer.styleLayer.id)) {
```

Comment:
is this function still used (isDataLayerVisible)? if not can we remove?

Reply:
Nope! Removed